### PR TITLE
Unit test cleanup

### PR DESCRIPTION
**Pull request by a1astair**
EE-126

Deleting old unit tests. Made sure that all the unit tests are passing. Got the project-service spec in a workable state. Testing the getById function as that is what we use the project service for.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
     "karma-jasmine": "~1.1.1",
     "karma-jasmine-html-reporter": "^0.2.2",
     "node": "^8.16.0",
-    "protractor": "^6.0.0",
+    "protractor": "~5.3.0",
```
Comment (KitArmstrong) [inline package.json]: Dropping this below 6 introduces 2 new high level security warnings from NPM.

File: src/app/project/documents/documents-tab.component.ts [hunk 2]
```diff
@@ -455,7 +455,7 @@ export class DocumentsTabComponent implements OnInit, OnDestroy {
       pageNumber,
       this.tableParams.pageSize,
       this.tableParams.sortBy,
-      { documentSource: 'PROJECT' },
+      { documentSource: 'PROJECT', eaoStatus: 'Published' },
```
Comment (KitArmstrong) [inline src/app/project/documents/documents-tab.component.ts]: I am not 100% sure, but I believe this was removed as it was deemed to cause a bug.

Comment (a1astair) [reply src/app/project/documents/documents-tab.component.ts]: good catch. This was probably from a bad rebase on my end.

File: package.json (hunk 1 above)
Comment (a1astair) [reply package.json]: Oh dang :( yea it wasn't working for me on 6. I will try again tho

Comment (a1astair) [reply package.json]: I updated to 5.4.3 which seems to work. Also admin is on 5.3 so we could update that. I think 5.4.3 is the latest version based on their npm [site](https://www.npmjs.com/package/protractor)

Comment (KitArmstrong) [reply package.json]: Nice, this removes the high vulnerabilities.

**Pull request merged**